CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.14.0
+
+*BREAKING CHANGES*
+
+- Drop support for encodings other than UTF-8 and ASCII.
+- Removed `parser_console.dart` library.
```

Comment:
why do we need to remove this library?

Reply:
It provides abstractions that aren't really used. One can just pass in a `List<int>`.

Since we're going through the work to do a breaking change, I felt cleaning up the API was useful.

Reply:
SGTM